memgpt/constants.py

```diff
@@ -55,6 +55,9 @@
 CORE_MEMORY_PERSONA_CHAR_LIMIT = 2000
 CORE_MEMORY_HUMAN_CHAR_LIMIT = 2000
 
+# Function return limits
+FUNCTION_RETURN_CHAR_LIMIT = 2000

```

Comment:
There were a few ways to cap the message output (could compute based on some fraction % of `agent.context_window`), but I figured might as well do something simpler like set it to a max char length similar since we're already doing this for human/persona. There's a chance the math can still go wrong ("go wrong" = enter some sort of "summarize deadlock" with a 2000 long truncated http request result) if the context window is too low, but with 8k I don't think we should have this problem. Also we make this assumption for persona/human anyways and it's working fine.